prog/dftb+/lib_math/eigensolver.F90

```diff
@@ -29,10 +29,6 @@ module dftbp_eigensolver
 #:endif
 
 
-  !> Used to return runtime diagnostics
-  character(len=100) :: error_string
```

Comment:
Not clear why this was replaced with copies in each subroutine.

Reply:
To really eliminate all global variables (although I agree, that collision here is unlikely, but still). On the module level, only constants should occur.